lib/matplotlib/figure.py

```diff
@@ -1827,24 +1827,44 @@ def subplots_adjust(self, *args, **kwargs):
     def ginput(self, n=1, timeout=30, show_clicks=True, mouse_add=1,
                mouse_pop=3, mouse_stop=2):
         """
-        Blocking call to interact with the figure.
-
-        This will wait for *n* clicks from the user and return a list of the
-        coordinates of each click.
-
-        If *timeout* is zero or negative, does not timeout.
-
-        If *n* is zero or negative, accumulate clicks until a middle click
-        (or potentially both mouse buttons at once) terminates the input.
-
-        Right clicking cancels last input.
+        Blocking call to interact with a figure. This will wait for *n*
+        clicks from the user and return a list of the coordinates of each
+        click.
 
         The buttons used for the various actions (adding points, removing
         points, terminating the inputs) can be overriden via the
         arguments *mouse_add*, *mouse_pop* and *mouse_stop*, that give
         the associated mouse button: 1 for left, 2 for middle, 3 for
         right.
 
+        Parameters
+        ----------
+        n : int, optional
+            Number of mouse clicks to accumulate. If negative, accumulate
+            clicks until the input is terminated manually. Default is 1.
```

Comment:
the numpydoc formats puts the default on the first line:

```
n : int, optional, default: 1
```

I find the numpydoc "offical" version more readable.